pkg/query-service/app/server.go

```diff
@@ -100,13 +99,13 @@ func (s *Server) createHTTPServer() (*http.Server, error) {
 
 	var reader Reader
 
-	storage := os.Getenv("STORAGE")
+	storage := storageURI
```

Comment:
Why are we creating this new variable? Can `storageURI` be reused? Also, is it better to rename `storageURI`?

Reply:
Fixed to use `storage` directly.